src/gui/optionsdlg.cpp

```diff
@@ -77,13 +77,14 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     setModal(true);
 
     // Icons
-    m_ui->tabSelection->item(TAB_UI)->setIcon(GuiIconProvider::instance()->getIcon("preferences-desktop"));
+    m_ui->tabSelection->item(TAB_APPEARANCE)->setIcon(GuiIconProvider::instance()->getIcon("preferences-desktop"));
+    m_ui->tabSelection->item(TAB_BEHAVIOUR)->setIcon(GuiIconProvider::instance()->getIcon("gear"));
```

Comment:
Or, no! The icon "preferences-desktop-theme" is for the "Appearance" tab. "preferences-desktop" should be used for the "Behaviour" tab as it is now. Things get complicated :) If you like, leave it as is.